ecr/Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM contentgroup/cqf-ruler:latest
+FROM localhost:5000/alphora/cqf-ruler:cqf-ruler-vsm-base
```

Comment:
should we use a shell script to rewrite this dockerfile every build to make it clear the local registry part is specific to our build process? Or is the fact that these changes are on the vsm_operations branch sufficient to make that distinction?